frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx

```diff
@@ -1,28 +1,30 @@
-# scExpression Documentation
+# scExpression — Query Gene Expression Across Tissues
+
+scExpression is a tool that allows users to query the expression of any gene across all data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```

Comment:
```suggestion
scExpression is a tool that allows users to query the expression of any gene across data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```

Reply:
@MDunitz would this be better? I don't think I'd want to get into many details in this intro sentence